rotkehlchen/tests/conftest.py

```diff
@@ -138,3 +143,49 @@ def requires_env(allowed_envs: list[TestEnvironment]):
         'CI' in os.environ and env not in allowed_envs,
         reason=f'Not suitable envrionment {env} for current test',
     )
+
+
+def get_cassette_dir(request: pytest.FixtureRequest) -> Path:
+    """
+    Directory structure for cassettes in each test file resembles the file's path
+    e.g. for tests in           `tests/unit/decoders/test_aave.py`
+         cassettes are in   `cassettes/unit/decoders/test_aave/`
+    """
+    return Path(request.node.path).relative_to(TESTS_ROOT_DIR).with_suffix("")
+
+
+@pytest.fixture
+def vcr_config(request: pytest.FixtureRequest, default_cassette_name: str) -> dict[str, Any]:
+    """
+    vcrpy config
+    - record_mode: allow rewriting multiple cassettes using CASSETTE_REWRITE_PATH env variable
```

Comment:
We can probably set this environment variable in CI using a github action that parses the commit message for this information. I noticed a similar pattern in `rotki/action-job-checker`.

cc @kelsos

Reply:
I would need to update the action-job-checker to do so but it is doable. 
Can you tell me what exactly do you need?

Reply:
For now I have updated the config to set `CASSETTE_REWRITE_PATH` by looking for a single line in the commit message that specifies the value (similar to how it would be set in bash):

```
begin multiline commit message
CASSETTE_REWRITE_PATH=unit/decoders
end multiline commit message
```

Reply:
This works, but it's very rudimentary, may not be the preferred approach across the team and also lacks validation.

So I'm happy with any approach that gets the result that we currently see in the config

Reply:
Yes, that is correct.

Reply:
If this is just to delete the cached cassettes then we can just delete the cached data from the actions UI